src/main/kotlin/apply/domain/evaluationItem/EvaluationItem.kt

```diff
@@ -9,19 +9,19 @@ import javax.persistence.Id
 @Entity
 class EvaluationItem(
     @Column(nullable = false)
-    val title: String,
+    val evaluationId: Long,
 
     @Column(nullable = false)
-    val description: String,
+    val title: String,
 
     @Column(nullable = false)
-    val evaluationId: Long,
+    val maximumScore: Int = 10,
```

Comment:
외부에서 필수로 값을 넣기 때문에 기본값을 가질 필요는 없다고 생각해요.

```suggestion
    val maximumScore: Int,
```

Reply:
입력 창에서부터 not null로 검증되니 필수로 입력된다고 보면 되겠군요